lib/bridge-client/index.js

```diff
@@ -19,8 +19,11 @@ var EventEmitter = require('events').EventEmitter;
 var UploadState = require('./upload-state');
 var Blacklist = require('./blacklist');
 var stream = require('readable-stream');
+var Readable = require('stream').Readable;
```

Comment:
We already depend on the `readable-stream` package, so let's use that for a stable streams base